docs/gencertchain.md

```diff
@@ -0,0 +1,18 @@
+# gencertchain
+
+A command-line tool for generating Certificate-Chain cbor file.
+
+# Usage
+
+This command generates an Certificate-Chain cbor file.
+
+```bash
+$ gencertchain -certificate ./prime256v1.pem > certchain.cbor
+```
+# Options
+
+- `-help`:                 Show help message.
+- `-ocsp` _string_:        Specify a DER-encoded OCSP response file. If omitted, it is automatically fetched from CA.
```

Comment:
nit: OCSP responder can be VA.
"...fetched from the certificate's OCSP reponder" could be getter?

Reply:
done.